.claude/skills/triage/scripts/apply.py

```
@@ -0,0 +1,96 @@
+    for verdict in json.load(open(args.validated)):
+        num = verdict["issue"]
+        if (only and num not in only) or num in skip:
+            continue
+        live = json.loads(
+            gh("issue", "view", str(num), "--repo", args.repo, "--json", "labels,state")
+        )
+        if live["state"] != "OPEN":
+            print(f"#{num} skipped: no longer open")
+            time.sleep(1)
+            continue
+        live_labels = {l["name"] for l in live["labels"]}
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)